apps/api/v1/test/lib/attendees/_post.test.ts

```diff
@@ -84,7 +86,8 @@ describe("POST /api/attendees", () => {
         timeZone: "UTC",
         bookingId: 1,
       };
-
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-expect-error
```

Comment:
These were existing errors that started showing up after enabing type-check in CI for api/v1